webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -174,8 +174,8 @@ export default class SearchableChannelList extends React.PureComponent<Props, St
         );
 
         const joinViewChannelButtonClass = classNames('btn', {
-            outlineButton: this.isMemberOfChannel(channel.id),
-            primaryButton: !this.isMemberOfChannel(channel.id),
+            'btn btn-secondary btn-sm outlineButton': this.isMemberOfChannel(channel.id),
+            'btn btn-primary btn-sm primaryButton': !this.isMemberOfChannel(channel.id),
```

Comment:
`btn` class is already added in the first argument of `classNames`. `btn-sm` seems to be also common (since one of the two options will always be selected). So, unless order matters, this could be rewritten to something like:

```
const joinViewChannelButtonClass = classNames('btn btn-sm', {
            'btn-secondary outlineButton': this.isMemberOfChannel(channel.id),
            'btn-primary primaryButton': !this.isMemberOfChannel(channel.id),
});